bindings/python/manifold3d.cpp

```diff
@@ -77,25 +113,26 @@ PYBIND11_MODULE(manifold3d, m) {
         ":param radius: For a given radius of circle, determine how many "
         "default");
 
-  py::class_<Manifold>(m, "Manifold")
-      .def(py::init<>())
-      .def(py::init([](std::vector<Manifold> &manifolds) {
-             Manifold result;
-             if (manifolds.size() >= 1) {
-               // for some reason using Manifold() as the initial object
-               // will cause failure for python specifically
-               // unable to reproduce with c++ directly
-               Manifold first = manifolds[0];
-               for (int i = 1; i < manifolds.size(); i++) first += manifolds[i];
-               return first;
-             } else {
-               return Manifold();
-             }
-           }),
-           "Construct manifold as the union of a set of manifolds.")
-      .def(py::self + py::self, "Boolean union.")
-      .def(py::self - py::self, "Boolean difference.")
-      .def(py::self ^ py::self, "Boolean intersection.")
+  nb::class_<Manifold>(m, "Manifold")
+      .def(nb::init<>())
+      .def(
+          "__init__",
+          [](Manifold *self, std::vector<Manifold> &manifolds) {
+            if (manifolds.size() >= 1) {
+              // for some reason using Manifold() as the initial object
+              // will cause failure for python specifically
+              // unable to reproduce with c++ directly
```

Comment:
I wonder if this was a pybind11 bug? Does it still repro?

Reply:
oh I did not check this